src/components/HendelseTypeFilter.tsx

```diff
@@ -138,26 +147,33 @@ export const HendelseTypeFilter = ({ personRegister }: Props): ReactElement => {
 const genererHendelseCheckbokser = (
   elementer: CheckboksElement[],
   onCheckedChange: (klikketElement: CheckboksElement, checked: boolean) => void,
+  isSendingAvForhandsvarselEnabled: boolean,
   personRegister?: PersonregisterState
 ) => {
-  return elementer.map((checkboksElement) => {
-    const filter = enkeltFilterFraTekst(checkboksElement.tekst, true);
-    const antall = Object.keys(
-      filterOnPersonregister(personRegister || {}, filter)
-    ).length;
-    const labelNode = (
-      <div>
-        {checkboksElement.tekst} <strong>({antall})</strong>
-      </div>
-    );
-    return (
-      <Checkbox
-        label={labelNode}
-        checked={checkboksElement.checked}
-        id={checkboksElement.key}
-        key={checkboksElement.key}
-        onChange={(e) => onCheckedChange(checkboksElement, e.target.checked)}
-      />
-    );
-  });
+  return elementer
+    .filter(
+      (checkboksElement) =>
```

Comment:
Vet vi har pleid å gjøre det sånn, men synes filtreringen med det uttrykket kan bli litt slitsomt å lese. Kanskje vi kan se på å forbedre det i en egen PR. Mulig man kunne man lagt til en `isEnabled`/`show`-property på CheckboksElement eller noe sånt.

Reply:
Ja, var litt usikker på hva som ble best. Og jeg satt noen minutter for å finne ut hva som ble riktig logikk her 🙃